### PR TITLE
{bio}[foss/2021b] SCGid v0.9b0

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)
Merge first:
* https://github.com/easybuilders/easybuild-easyblocks/pull/2688
* https://github.com/easybuilders/easybuild-framework/pull/3976

### CODE REVIEW

**Comment by casparvl** [comment]
Thanks for your contribution @ItIsI-Orient ! It seems that your patch file did not include any description, which is why the CI failed. Can you please include a description at the top of your patch file to indicate what the issue was that was fixed? If there is a github issue or PR on it in the SCGid repo, it's worth putting a link to those in the description too. That way, it's also easier to check for future versions of this EasyConfig if the patch is still relevant or not. As an example, you can have a look at [this patch](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/PyTorch/PyTorch-1.10.0_fix-alias-violation-in-bitwise-ops.patch)

**Comment by ItIsI-Orient** [comment]
@casparvl Hi, yes, I know about this issue and that is already fixed in my current working version. I will push a fix into this PR probably tomorrow, but I will still have to wait until my framework PR gets reviewed and approved, since one of the dependencies needed (Databionics ESOM) has to be installed and provided by user and therefore cannot be found by easybuild.

**Comment by casparvl** [comment]
Ah, sorry, I overlooked the part that this depends on another PR. I know the maintainer who's keeping an eye on `easybuild-framework` has not found a lot of time to work on PRs this week, so I guess we just have to be patient :)

File: easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb
Comment (boegel) [inline easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb:52]: These should be removed, they don't actually do anything w.r.t. the installation

File: easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb
Comment (boegel) [inline easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb:46]: please use `/python%(pyshortver)s/`

or just remove, `local_config_path` is not used anymore

File: easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb
Comment (boegel) [inline easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb:47]: this can be removed too

File: easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb
Comment (boegel) [inline easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb:28]: `Python` should be a runtime dependency?

File: easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb
Comment (boegel) [inline easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb:59]: This should be renamed on download, try this:

```python
'sources': [{
    'download_filename': 'e326c34307803b9ea8ffc975ddd9bd05643af931.tar.gz',
    'filename': 'ClaMS-CLI-%(version)s.tar.gz',
}],

File: easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-initEdit.patch
Comment (boegel) [inline easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-initEdit.patch:11]: provide a fallback value to avoid hard crash:

```python
os.getenv("EBROOTSCGID", "SCGID_INSTALL_DIR_UNKNOWN")

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb
Comment (ItIsI-Orient) [reply easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb:68]: @boegel Oh, we want user to name the Tarball `esom-1.1.tar.gz`? Okay, will do

File: easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb
Comment (boegel) [inline easybuild/easyconfigs/s/SCGid/SCGid-0.9b0-foss-2021b.eb:58]: @ItIsI-Orient Final request for change: I would tweak these instructions a bit, to make them clearer (less room for interpretation).

Done in https://github.com/ItIsI-Orient/easybuild-easyconfigs/pull/1

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2615.swalot.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/4211e1a890e25308ec28d59768c245a6 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3519.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/faea80ddbabebeb6774cb4441180d2e7 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**
